docs-release-notes.md

```diff
@@ -1,3 +1,22 @@
+# April 12, 2024
+
+## Edge Server
+
+- Edge Server MongoDB API Support: Add `findAndModify` command to `update` options.
```

Comment:
This hasn't been published yet - it was merged to a feature branch - so we should remove this from the release notes.